src/assets/i18n/af.json

```
@@ -506,7 +506,7 @@
   "Browse to the keytab file to upload.": "",
   "Browse to the locally saved configuration file.": "",
   "Browse to the operating system installer image file.": "",
-  "Browse to the path to be copied. FreeBSD  file path limits apply. Other operating systems can have different  limits which might affect how they can be used as sources or  destinations.": "",
+  "Browse to the path to be copied. Linux  file path limits apply. Other operating systems can have different  limits which might affect how they can be used as sources or  destinations.": "",
```

Found these redundant empty spaces. I see the same in the other i18n files - is this an error from translation file generation?
```suggestion
  "Browse to the path to be copied. Linux file path limits apply. Other operating systems can have different limits which might affect how they can be used as sources or destinations.": "",
```

---

It's an artifact of how the code was written.